docs-js/orchestration/chat-completion.mdx

```diff
@@ -0,0 +1,643 @@
+---
+id: chat-completion
+title: Chat Completion
+hide_title: false
+hide_table_of_contents: false
+description: How to use the SAP Cloud SDK for AI to perform chat completion tasks using orchestration service from SAP AI Core.
+keywords:
+  - sap
+  - cloud
+  - sdk
+  - ai
+  - orchestration
+  - chat
+  - completion
+---
+
+The `@sap-ai-sdk/orchestration` package provides a client for the orchestration service of SAP AI Core.
+The orchestration service harmonizes the API of various generative AI models, enabling seamless integration and interaction with different models through a unified interface.
+Additionally, it provides features like [templating](#templating), [content filtering](#content-filtering), [grounding](#grounding) and many more to enhance the interaction with generative AI models.
+
+Find more details about orchestration workflow [here](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/orchestration-workflow).
+
+## Installation
+
+```
+$ npm install @sap-ai-sdk/orchestration
+```
+
+## Usage
+
+Initialize a client with proper orchestration configuration for each modules.
+Below is an example for configuring the mandatory `llm` and `templating` modules.
+
+```ts
+import { OrchestrationClient } from '@sap-ai-sdk/orchestration';
+
+const orchestrationClient = new OrchestrationClient({
+  llm: {
+    model_name: 'gpt-4o'
+  },
+  templating: {
+    template: [
+      { role: 'user', content: 'Hello World! Why is this phrase so famous?' }
+    ]
+  }
+});
+```
+
+Here we specify the model name with a user message as prompt.
+
+To send a chat completion request, use the `chatCompletion()` method.
+Use the following convenience methods for handling chat completion response:
+
+- `getContent()` parses the response and returns the model's output as a string.
+- `getFinishReason()` retrieves the `finish_reason` explaining why chat completion request stopped.
+- `getTokenUsage()` provides token usage details, including `total_tokens`, `prompt_tokens`, and `completion_tokens`.
+
+```ts
+const response = await orchestrationClient.chatCompletion();
+
+console.log(response.getContent());
+console.log(response.getFinishReason());
+console.log(JSON.stringify(response.getTokenUsage()));
+```
+
+In the following sections, we explain the configuration for each module in detail.
```

Comment:
I feel like we dont need these lines here anymore. With docusaurus we have links displayed on the right which clearly shows docs structure. I would also add the sample code link somewhere else (maybe at the top)

Reply:
Currently heading starting from the third level are not displayed on the right sidebar. Not sure if we can configure this.

Reply:
I moved the sample code link before the first example.